openshift/README.md

```diff
@@ -1,33 +1,49 @@
-# Deploying fabric8-analytics services
+# Openshift green field deployment of fabric8-analytics services
 
-## Create a new project
+Config map and secrets are generated from the configuration stored in environment variables.
 
-`oc login` to your favorite OpenShift cluster and create a new project there:
+## Configure fabric8-analytics services
+All configuration for the deployment script resides in env.sh.
```

Comment:
Maybe we could provide some `env.sh-template` file which would contain all env. variables that are needed, possibly with some comment briefly explaining where to get the values from?

Reply:
And then we could instruct people to do just `cp env.sh-template env.sh` and then modify the `env.sh`.